src/components/PasswordField/index.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable react/prop-types */
+import React, { Component, PropTypes as T } from 'react';
+import { blissComponent } from 'decorators';
+import Btn from 'components/Btn';
+
+@blissComponent
+export default class PasswordField extends Component {
+
+	static propTypes = {
+		icon: T.string,
+		input: T.object,
+	}
+
+	static defaultProps = {
+		icon: 'eye',
+	}
+
+	constructor(props) {
+		super(props);
+		this.state = {
+			show: false,
+			stateClass: '',
+		};
+		this.handleToggleShowPassword = this.handleToggleShowPassword.bind(this);
+	}
+
+	handleToggleShowPassword() {
+		this.setState({
+			show: !this.state.show,
```

Comment:
tady to jsou duplicitni informace, melo by tu byt jenom `show` a tu classu dopocitat pri renderu